k8s/namespaces/fees-pay/ccpay-payment-api/perftest.yaml

```diff
@@ -1,21 +1,11 @@
 apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
-  name: ccpay-payment-app
-  namespace: fees-pay
+  name: ccpay-payment-api
 spec:
   releaseName: ccpay-payment-api
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/payment-api
   values:
     java:
-      replicas: 2
-      useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/payment/api:pr-1150-b7e864c-20220316113852 #{"$imagepolicy": "flux-system:perftest-ccpay-payment-app"}
```

Comment:
Review image policy naming, can it match HelmRelease?

Reply:
resolved